SPECS/gpumon.spec

```diff
@@ -13,7 +13,6 @@ Source1: SOURCES/gpumon/xcp-rrdd-gpumon.service
 Provides: gitsha(https://code.citrite.net/rest/archive/latest/projects/XSU/repos/gpumon/archive?at=v0.18.0&format=tar.gz&prefix=gpumon-0.18.0#/gpumon-0.18.0.tar.gz) = 23164e42d7a466475e67e98046f4be647b6d790d
 
 BuildRoot:      %{_tmppath}/%{name}-%{version}-root
-BuildRequires:  gdk-devel
```

Comment:
Arguable, but since we still sync that spec file from CH, I'd add a comment explaining why we remove the BuildRequires, and comment the "BuildRequires" line instead of deleting it.